src/RRule.php

```diff
@@ -738,7 +738,7 @@ public function occursOn($date)
 	 */
 	public function occursAt($date)
 	{
-		$date = self::parseDate($date);
+		$date = self::parseDate($date)->setTimezone($this->dtstart->getTimezone());
```

Comment:
I think this will create problems because you are overriding the timezone if one was provided. You should only set the timezone only if `$date` does not already contain a timezone.